Guides/RepeatWords/Playing.js

```diff
@@ -8,14 +8,12 @@ export default class Playing extends Component {
 
         this.state = {
             wordsOpacity: new Animated.Value(0),
-            progressValueListener: null,
-            progressCircleAnimationProcess: null,
         }
     }
 
     componentDidMount() { this.startInstructionsTextAnimation(); }
 
-    componentWillUnmount() { Animated.timing(this.state.wordsOpacity).stop(); }
+    componentWillUnmount() { Animated.sequence(this.state.wordsOpacity).stop(); }
```

Comment:
What’s the difference between `timing` and `sequence`?

Reply:
`sequence` takes an array of `timing` animations. This `stop()` actually stops all of the animations. Before it didn't.